kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -5501,8 +5501,12 @@ messages:
             if IsClass(oJalaSpell,&Restorate)
             {
                iJalaPower = Nth(Nth(lJalaInfo,3),3);
+
+               iTime = bound(iTime,1000,60000);
                iTime = Send(oJalaSpell,@AdjustHealthTime,#time=iTime,
                             #iSpellPower=iJalaPower);
+
+               return bound(iTime,670,60000);
```

Comment:
Can these changes to player.kod be moved into Restorate instead?  Player is already enormous so it's good to keep new code out of it.  Plus these additional magic numbers seem to be specific to Restorate.

Reply:
I pushed both bounding statements over to Restorate.kod.  We can do this in fewer lines but I think this is readable.

I thought about going for return bound(Send(oJalaSpell,@AdjustHealthTime,#time=iTime,#iSpellPower=iJalaPower); or even bounding the iTime argument in the function call but I think clever code can be problematic down the road.  Although _return bound(send_ is used in one other spot in the database.